lib/stringToChars.js

```diff
@@ -0,0 +1,11 @@
+
+function stringToChars(string) {
+  let arrayChars = [];
+  for (let i of string) {
```

Comment:
Be careful with a variable name like `i` here. 99% of programmers will believe a variable named `i` is meant to be an index into a list.

Something like:

```javascript
for (let char of string) {
  // ..
}
```

is clearer.